packages/extension/src/background/networkStatus.ts

```diff
@@ -13,7 +13,7 @@ const swrStorage = new KeyValueStorage<Record<SwrCacheKey, any>>(
 // see: https://github.com/jperasmus/stale-while-revalidate-cache#configuration
 const swr = createStaleWhileRevalidateCache({
   storage: swrStorage, // can be any object with getItem and setItem methods
-  minTimeToStale: 60e3, // 1 minute
+  minTimeToStale: 2 * 60e3, // 2 minutes
   maxTimeToLive: 30 * 60e3, // 30 minutes
 })
```

Comment:
This will not fire API in the background. The wallet UI queries this periodically when the wallet is open, and `minTimeToStale` determines how long the network status is cached in background before it does the network requests again. Now it is set to 2 minutes instead.